db.go

```diff
@@ -161,6 +167,14 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		Name: "prometheus_tsdb_tombstone_cleanup_seconds",
 		Help: "The time taken to recompact blocks to remove tombstones.",
 	})
+	m.storageBytes = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "prometheus_tsdb_storage_bytes",
+		Help: "The number of bytes that are currently used for local storage.",
+	})
+	m.dataLimitDeletions = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_tsdb_data_limit_deletions",
```

Comment:
Hmm, how about `prometheus_tsdb_data_limit_deletions` -> `prometheus_tsdb_size_limit_deletions`?

Reply:
(nit) a `_total` suffix is expected too.